pkg/types/service_offering.go

```diff
@@ -63,6 +63,11 @@ func (e *ServiceOffering) Validate() error {
 	if e.BrokerID == "" {
 		return fmt.Errorf("service offering broker id missing")
 	}
+	if cliFriendly, err := util.IsCLIFriendly(e.CatalogName); err != nil {
+		return err
+	} else if !cliFriendly {
+		return fmt.Errorf("service offering catalog name is not CLI-friendly")
```

Comment:
can you extract this error and include what actually CLI-friendly means in the response.